content/en/tracing/profiler/enabling/node.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Enabling the Node.js Profiler
+kind: Documentation
+code_lang: node
+type: multi-code-lang
+code_lang_weight: 40
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'tracing/profiler/search_profiles'
+      tag: 'Documentation'
+      text: 'Learn more about available profile types'
+    - link: 'tracing/profiler/profiler_troubleshooting'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+{{< site-region region="us5" >}}
+<div class="alert alert-warning">
+  The Continuous Profiler is not available for the Datadog {{< region-param key="dd_site_name" >}} site.
+</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">
+Datadog Node.js Profiler is currently in public beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The profiler is shipped within Datadog tracing libraries. If you are already using [APM to collect traces][1] for your application, you can skip installing the library and go directly to enabling the profiler.
+
+## Requirements
+
+The Datadog Profiler requires Node.js 12+, but Node.js 16+ is recommended. Some applications will see tail latency spikes every minute while starting the next profile on Node.js < 16. 
+
+## Installation
+
+To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][2]+ or [6.20.2][3]+.
+
+2. Run `npm install --save dd-trace@latest`. This adds a dependency on the `dd-trace` module which includes the profiler.
+
+3. You can auto-enable the profiler with environment variables:
+
+    ```shell
+    export DD_PROFILING_ENABLED=true
+    export DD_ENV=prod
+    export DD_SERVICE=my-web-app
+    export DD_VERSION=1.0.3
+    ```
+
+    or in code:
+
+    ```js
+    const tracer = require('dd-trace').init({
+      profiling: true,
+      env: 'prod',
+      service: 'my-web-app',
+      version: '1.0.3'
+    })
+    ```
+
+    **Note**: If you’re already using Datadog APM, you should already be calling `init` and don’t need to do so again. If you are not, ensure the tracer and the profiler are loaded together:
+
+    ```node
+    node -r dd-trace/init app.js
+    ```
+
+    or
+
+    ```node
+    const tracer = require('dd-trace/init')
+    ```
+
+4. A minute or two after starting your Node.js application, your profiles will show up on the [Datadog APM > Profiler page][4].
+
+## Not sure what to do next?
```

Comment:
We should add another section here that says something along the lines of:

### Experiencing High Overhead?

Node.js 16+ is recommended. Some applications will see tail latency spikes every minute while starting the next profile on Node.js < 16.